### PR TITLE
reexport mean and median

### PR DESCRIPTION
**Pull request by milesfrain**

### CODE REVIEW

File: src/BenchmarkTools.jl [hunk 1]
```diff
@@ -2,11 +2,8 @@ module BenchmarkTools
 
 using JSON
 using Base.Iterators
-
-using Statistics
```
Comment (ararslan) [inline src/BenchmarkTools.jl]: We still want to be `using Statistics` here and extending the functions from Statistics; we can simply add `mean` and `median` to the exports, which will reexport the right things.

Comment (milesfrain) [reply src/BenchmarkTools.jl]: Good suggestion. Updated latest PR.

File: src/BenchmarkTools.jl [hunk 2]
```diff
@@ -46,6 +48,8 @@ export BenchmarkGroup,
        invariants,
        regressions,
        improvements,
+       median,
```
Comment (ararslan) [inline src/BenchmarkTools.jl]: Listed twice?

Comment (milesfrain) [reply src/BenchmarkTools.jl]: I assumed a separate export was need to for both `::Trial` and `::BenchmarkGroup`, but with the magic of Julia, it seems like only one is necessary. Will update to list these only under Trial.

**Pull request merged**